code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -306,6 +306,11 @@
 /obj/spacepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
 	to_chat(S, "<span class='warning'>Destroying this vehicle would destroy us. Aborting.</span>")
 
+/obj/machinery/clonepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	if(attempting)
```

Comment:
why not check for `occupant`?

Reply:
Please make the change within two days or this PR shall be closed for staleness.